shared/haml/user_header.haml

```diff
@@ -25,21 +25,22 @@
   end
 
 - if show_create_menu
-  .header_button.create_menu{tabindex: 0, role: "button"}
+  .header_button.create_menu{id: "header_create_menu", tabindex: 0, role: "button"}
     %span.create_button
       = I18n.t("#{loc_prefix}create")
     &nbsp;
     %i.create_menu_arrow_down{class: "fa fa-caret-down"}
     %i.create_menu_arrow_up{class: "fa fa-caret-up", style: "display: none"}
     .create_options{style: 'display: none', dir: language_dir_class}
       - create_drop_down_items.each do |entry|
-        %a.project_link_box{href: entry[:url]}
+        %a.project_link_box{id: "create_menu_option_#{entry[:title]}", href: entry[:url]}
           %img{src: "/shared/images/fill-70x70/courses/#{entry[:image]}"}
           .project_link{ id: entry[:id]}
             .text= I18n.t("#{loc_prefix}#{entry[:title]}")
-      %a.project_link_box{href: CDO.studio_url('projects')}
+      %a.project_link_box{id: "create_menu_option_view_all", href: CDO.studio_url('projects')}
         .project_link#view_all_projects
           = I18n.t("#{loc_prefix}view_all")
+    = render inline: File.read(Rails.root.join('..', 'shared', 'haml', 'create_menu_event_logger.haml')), type: :haml, locals: {is_signed_out: current_user.nil?, options: create_drop_down_items.map{|entry| entry[:title]}.append("view_all")}
```

Comment:
I used [this line](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/layouts/application.html.haml#L119) as reference for how to render a `.haml` file that's in the `/shared` directory. It wasn't working when I tried something simpler like `= view :create_menu_event_logger`, I think because it was expecting something in a `views` directory (which `/shared` doesn't have).